### PR TITLE
Registration expert advice

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/learn/expert-advice/identity-basics/registration-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+layout: advice
+title: Best practices for registration forms
+description: Users have to register to gain application access. How can you make this as painless as possible?
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/registration-best-practices.md:4]: "Users have to fill out registration forms in order to gain application access"... or some way to work in 'registration forms'

File: site/learn/expert-advice/identity-basics/registration-best-practices.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+category: Identity Basics
+date: 2020-09-08
+dateModified: 2020-09-08
+---
+
+Signing up for accounts is something we're all familiar with. It's a gateway to applications we want and need. But it's not really fun. Or even pleasant. After all, we're signing up to access the application, not because we want to set up another username and password and enter personal data into yet another system:
+
+{% include _image.liquid src="/assets/img/advice/long-registration-form.png" alt="A long registration form. Honestly? You need my full name and my middle name?" class="img-fluid" figure=false %}
+
+You can make the registration process simpler. And you should.
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/registration-best-practices.md:16]: "you can make the registration form process"

File: site/learn/expert-advice/identity-basics/registration-best-practices.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+dateModified: 2020-09-08
+---
+
+Signing up for accounts is something we're all familiar with. It's a gateway to applications we want and need. But it's not really fun. Or even pleasant. After all, we're signing up to access the application, not because we want to set up another username and password and enter personal data into yet another system:
+
+{% include _image.liquid src="/assets/img/advice/long-registration-form.png" alt="A long registration form. Honestly? You need my full name and my middle name?" class="img-fluid" figure=false %}
+
+You can make the registration process simpler. And you should. 
+
+## Is registration needed at all?
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/registration-best-practices.md:18]: Are registration forms needed at all?

File: site/learn/expert-advice/identity-basics/registration-best-practices.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+Provide clear error messages when data fails to validate. Use both client side validation, which is faster, and server side validation, which is tamper proof. On the topic of tampering, ensure any form is submitted over TLS. You want to keep submitted information confidential and secure.
+
+Make use of the full suite of HTML elements. Dropdowns and radio buttons are powerful, but number and email input fields leverage browsers' built-in validation and should be used as well. If you aren't sure what's supported, use tools like [caniuse.com](https://caniuse.com/) to verify compatibility.
+
+### Registration forms are unique
+
+But registration forms aren't just another form. They are the gateway to your full application or site.
+
+What causes angst when a user is signing up? This [2012 paper (PDF)](https://discovery.ucl.ac.uk/id/eprint/1378346/1/ewic_hci12_diss_paper7.pdf) examined registration for government services. It defined sign up friction as "as the imbalance between the business process (user goals) and [required] security behaviour" around signing up. This study found friction was best explained by the following attributes of a signup process:
```
Comment (dpolstra) [inline site/learn/expert-advice/identity-basics/registration-best-practices.md:80]: It defined sign up friction as "~as~ the imbalance between..."

Comment (mooreds) [reply site/learn/expert-advice/identity-basics/registration-best-practices.md:80]: Awesome, thanks!

**Pull request merged**